spec/controllers/api/collectivities_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+describe Api::CollectivitiesController, type: :controller do
+  describe "GET index" do
+    let(:collectivities) { create_list(:collectivity, 5) }
+
+    before do
+      collectivities
+    end
+
+    it "returns the collectivities" do
+      get :index
+      body = response.parsed_body
+
+      expect(body.pluck("siret")).to eq collectivities.map(&:siret)
```

Comment:
```suggestion
      expect(body.pluck("siret")).to eq collectivities.map(&:siret)
      expect(body.count).to eq(5)
```

Tu pourrais d'ailleurs n'en créer que 2 (au lieu de 5)